docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -1,14 +1,16 @@
 # Development Environment
 
-We will need a place to work and tools for creating a block, this is often referred to as the development environment. The three main pieces needed for our development environment are:
+The development environment setup guide is for setting up your local development environment for JavaScript development; creating plugins and tools for extending WordPress and the block editor.
+
+A development environment is a catch-all term for the setup on your computer to work. The three main pieces needed for our development environment are:
 
 1. Node/NPM Development Tools
 2. WordPress Development Site
 3. Code Editor
 
```

Comment:
**The short guide.**

In terminal - Mac, Windows or Linux.
Install nvm:
`curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash`
Quit and restart terminal.
Install node:
`nvm install --lts`

Download and install Docker. Startup Docker (?) ...

In terminal.
`npm -g install @wordpress/env`
Check to see it has been installed:
`wp-env --version`

Do something (?)
Then in terminal:
`wp-env start`

You should now have a local development environment in place. If something does not work then check the uninstall guide to remove and start over. - @mkaz and @noahtallen  it would be good to have a section for uninstalling all that we have installed. To start over if needed.

Reply:
The do something step is from the Create a Block tutorial, on generating the plugin directory see [wp-plugin.md](https://github.com/WordPress/gutenberg/blob/master/docs/designers-developers/developers/tutorials/create-block/wp-plugin.md). 

```
npx @wordpress/create-block gutenpride
cd gutenpride
wp-env start
```

The create block script creates a plugin directory, you run the wp-env start from within that directory.

Reply:
you can uninstall wp-env like so:
- run `wp-env destroy` to destroy the local environment data for a specific projest
- run `npm uninstall -g @wordpress/env` will uninstall the npm package, but will not remove any environment data

Reply:
The do something step is confusing as we are having nice and logical steps on what to do in terminal to setup a local WordPress development. I expect the do something step will then create the local WordPress development site which we can then easily access. I am saying the same thing just below. We should not refer to another tutorial as that information can come later.

Reply:
Thanks Noah! Btw how do we test to see that all of it is actually uninstalled?
Having uninstall information and checking to see that it is uninstalled would be helpful.

Reply:
If it is actually uninstalled, then `wp-env start` will result in an error like "command not found". Additionally, if you destroyed your environment, the command `docker ps -a`, which shows a list of local environments, will not display environments associated with the local environment that was deleted.

Verifying that the environment was properly uninstalled is kind of complicated because you have to understand how Docker works and how wp-env works under the hood, since several different things are cleaned up when you run `wp-env destroy`

Reply:
I am thinking as long as one gets the error messages one is supposed to get then that would be the check that it is uninstalled.